src/selectedtests/task_mappings/mappings.py

```diff
@@ -160,47 +171,45 @@ def _get_evg_project_and_init_repo(
     )
 
 
-def _get_filtered_files(diff: DiffIndex, regex: Pattern) -> List[str]:
+def _get_filtered_files(diff: DiffIndex, regex: Pattern) -> Set[str]:
```

Comment:
I changed the files we are looking at to use `Set` instead of `List` since the files should always be unique and we don't care about order.